src/caveexpress/server/entities/PackageTarget.cpp

```diff
@@ -49,9 +49,7 @@ void PackageTarget::onPreSolve (b2Contact* contact, IEntity* entity, const b2Man
 		setAnimationType(Animations::ANIMATION_ACTIVE);
 		_package = package;
 		
-		b2Vec2 p = getPos();
```

Comment:
This is bad in multiplayer games where multiple players can deliever multiple packages at the same time. Globals sounds without position and wirhout fading-by-distance would not work here.